sdk/communication/azure-communication-chat/azure/communication/chat/_shared/utils.py

```diff
@@ -66,12 +65,37 @@ def create_access_token(token):
     try:
         padded_base64_payload = base64.b64decode(parts[1] + "==").decode('ascii')
         payload = json.loads(padded_base64_payload)
-        return AccessToken(token,
-            _convert_expires_on_datetime_to_utc_int(datetime.fromtimestamp(payload['exp']).replace(tzinfo=TZ_UTC)))
```

Comment:
AccessToken expects expires_on to be an int. Why is this being removed?

This the definition of AccessToken:
https://github.com/Azure/azure-sdk-for-python/blob/621efa339d0e911c64077291bd0d730c0ecde45f/sdk/core/azure-core/azure/core/credentials.py#L14

Reply:
Thanks for your comment. I did not know they are being used since CI tests passed. I have reverted the change in the latest iteraiton.